po/pl.po

```diff
@@ -696,149 +787,153 @@ msgstr "Skopiuj do schowka"
 msgstr "Zapisz ustawienia"
 
-#: data/ui/preferences_window.blp:34
+#: data/ui/preferences_window.blp:58
 msgid "Image Format"
 msgstr "Format obrazu"
 
-#: data/ui/preferences_window.blp:35
+#: data/ui/preferences_window.blp:59
 msgid "Default format for saved screenshots"
 msgstr "Domyślny format zapisywanych zrzutów ekranu"
 
-#: data/ui/preferences_window.blp:39
-msgid "Compress image"
-msgstr "Skompresuj obraz"
-
-#: data/ui/preferences_window.blp:40
-msgid "On supported formats"
-msgstr "Wspierane formaty obrazu"
-
-#: data/ui/preferences_window.blp:41
-msgid "Compress images to reduce file size"
-msgstr "Skompresuj obraz, aby zredukować rozmiar pliku"
-
-#: data/ui/preferences_window.blp:46
-msgid "Confirm on close"
-msgstr "Potwierdź, aby zamknąć"
+#: data/ui/preferences_window.blp:71
+msgid "Confirm on Close"
+msgstr "Potwierdź przy zamknieciu"
 
-#: data/ui/preferences_window.blp:47 data/ui/preferences_window.blp:48
+#: data/ui/preferences_window.blp:72 data/ui/preferences_window.blp:73
 msgid "Ask for confirmation before closing the app"
 msgstr "Zapytaj przez zamknięciem programu"
 
-#: data/ui/preferences_window.blp:55
+#: data/ui/preferences_window.blp:79
 msgid "Screenshot Management"
 msgstr "Zarządzanie zrzutami ekranu"
 
-#: data/ui/preferences_window.blp:58
-msgid "Trash screenshots on close"
+#: data/ui/preferences_window.blp:82
+msgid "Trash Screenshots on Close"
 msgstr "Usuń zrzuty ekranu przy zamykaniu"
 
-#: data/ui/preferences_window.blp:59
+#: data/ui/preferences_window.blp:83
 msgid "Automatically move screenshots taken from within the app to trash"
 msgstr "Automatycznie przenieś zrzuty ekranu wykonane w programie do kosza"
 
-#: data/ui/preferences_window.blp:60
+#: data/ui/preferences_window.blp:84
 msgid "Trash screenshots on app close"
 msgstr "Usuń zrzuty ekranu przy zamykaniu programu"
 
-#: data/ui/preferences_window.blp:66
+#: data/ui/preferences_window.blp:90
 msgid "Image Upload Providers"
-msgstr "Dostawcy serwisów do wczytania obrazów"
+msgstr "Dostawcy serwisów do przesłania obrazów"
 
-#: data/ui/preferences_window.blp:67
+#: data/ui/preferences_window.blp:91
 msgid "Connect to your favorite upload providers, or create a link yourself"
 msgstr ""
-"Połącz z ulubionym serwisem do wczytania obrazów, lub samodzielnie utwórz "
+"Połącz z ulubionym serwisem do przesłania obrazów, lub samodzielnie utwórz "
 "łącze"
 
-#: data/ui/preferences_window.blp:71
+#: data/ui/preferences_window.blp:95
 msgid "Learn More..."
 msgstr "Dowiedz się więcej..."
 
-#: data/ui/preferences_window.blp:73
+#: data/ui/preferences_window.blp:97
 msgid "Learn more about providers"
 msgstr "Dowiedz się więcej o dostawcach"
 
-#: data/ui/preferences_window.blp:78
+#: data/ui/preferences_window.blp:102
 msgid "Provider"
 msgstr "Dostawca"
 
-#: data/ui/preferences_window.blp:88
-msgid "None Selected"
-msgstr "Nic nie wybrano"
+#: data/ui/preferences_window.blp:123
+msgid "Confirm on Upload"
+msgstr "Potwierdzenie przy przesłaniu"
```

Comment:
"przesłaniu" should be inflected to "przesyłaniu", because this is a confirmation that shows up before uploading, not after.